src/main/resources/assets/traverse/lang/zh_cn.json

```
@@ -3,20 +3,20 @@
 	"biome.traverse.autumnal_wooded_hills":"秋木山脉",
 	"biome.traverse.autumnal_woods":"秋木树林",
 	"biome.traverse.cliffs":"悬崖",
-	"biome.traverse.coniferous_forest":"针叶森林",
-	"biome.traverse.coniferous_woodland_hills_biome":"针叶林山",
+	"biome.traverse.coniferous_forest":"冷杉森林",
+	"biome.traverse.coniferous_woodland_hills_biome":"冷杉林山",
 	"biome.traverse.desert_shrubland":"沙漠灌丛带",
-	"biome.traverse.high_coniferous_forest":"高针叶森林",
+	"biome.traverse.high_coniferous_forest":"高冷杉森林",
 	"biome.traverse.lush_swamp":"茂密沼泽",
 	"biome.traverse.meadow":"牧场",
 	"biome.traverse.mini_jungle":"迷你丛林",
 	"biome.traverse.plains_plateau":"高地平原",
 	"biome.traverse.rocky_edge":"岩壁",
 	"biome.traverse.rolling_hills":"绵延山丘",
-	"biome.traverse.snowy_cliffs":"覆雪悬崖",
-	"biome.traverse.snowy_coniferous_forest":"覆雪针叶森林",
-	"biome.traverse.snowy_coniferous_wooded_hills":"覆雪针叶林山脉",
-	"biome.traverse.snowy_high_coniferous_forest":"覆雪高针叶森林",
+	"biome.traverse.snowy_cliffs":"积雪的悬崖",
+	"biome.traverse.snowy_coniferous_forest":"积雪的冷杉森林",
+	"biome.traverse.snowy_coniferous_wooded_hills":"积雪的冷杉林山脉",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"biome.traverse.snowy_coniferous_wooded_hills":"积雪的冷杉林丘陵",
```

Mountains 才是「山地」

Hills 一词在[沼泽变种](https://minecraft-zh.gamepedia.com/%E6%B2%BC%E6%B3%BD#.E6.B2.BC.E6.B3.BD.E5.B1.B1.E4.B8.98)处被翻译为「山丘」，但可能是翻译错误，建议联系广药或 ff。其它译名中都是「丘陵」。